cds/modules/theme/static/js/cds/module.js

```diff
@@ -310,6 +355,39 @@ app.filter('groupDownloadable', function() {
   });
 });
 
+// Sort video subformats descending by height
+app.filter('sortVideosDescending', function() {
+  return function(videos) {
+    return _.orderBy(videos, function(video) {
+      return parseInt(video.tags.height);
+    }, ['desc']);
+  }
+});
+
+// Gett all files with types passed as argument
+app.filter('getFilesByType', function() {
+  return function(files, types) {
+    if (!Array.isArray(files) || !Array.isArray(types) || !types.length || !files.length) { return files; }
```

Comment:
just a proposition: Isn't better you think to write the `!types.length` as `types.length == 0` just to be quicker to understand? I know it's the same but I am trying to think how is faster to understand it when you show it :) Also you can reduce this to just checking `types.length == 0 || files.length == 0` or if you don't want to change the way is written to `!types.length || !files.length`.

Reply:
@zzacharo yes I can but what if I receive 2 strings? as parameters. I can remove Array.isArray(files) but usually this are best practices to check if the parameter is actually an array.